pkgs/development/python-modules/libmambapy/default.nix

```diff
@@ -0,0 +1,65 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildPythonPackage,
+  cmake,
+  ninja,
+  libmamba,
+  pybind11,
+  setuptools,
+  wheel,
+}:
+buildPythonPackage rec {
+  pname = "libmambapy";
+  version = "1.5.7";
+  pyproject = true;
+
+  nativeBuildInputs = [
+    cmake
```

Comment:
I didn't try building, but wouldn't this require skbuild?

Reply:
Not that I noticed.
I see that it's a required package according to upstream but it wasn't needed here.